gulpfile.js

```diff
@@ -67,6 +69,16 @@ gulp.task('lib', function() {
 gulp.task('flow', function() {
   return gulp
     .src(paths.lib.src)
+    .pipe(gulpModuleMap(moduleMapOpts))
+    .pipe(gulpStripProvidesModule())
```

Comment:
I think Haste would resolve to this .flow file with providesModeul and then the requires within would be relative and still point to other modules which have a .flow file as well. I think Haste would use the `./` at the beginning of the requires here to drop out of pM resolution one it gets into a file. I think.

The problem is that we run flow check on the untransformed files in other projects. So until all of them only do `require('fbjs/whatever')`, then we have to make both work.